src/components/ActionButton/ActionButton.module.scss

```diff
@@ -7,3 +9,7 @@
   color: black;
   cursor: pointer;
 }
+
+.button:hover {
+  background-color: #0ae5f5;
```

Comment:
The action button was already the primary color and I believe it's good for buttons to indicate pliancy, is there better way to do it?

Reply:
The hover color itself is fine, just that maybe it should be turned into a variable in `themes.scss` (so that it can be used as e.g. `var(--primary-hover)`). You could also import `colors.scss` if you wish to just use the colors that are defined there, but just make sure to be consistent. The primary goal here is to have the actual hex color values defined in one file.

Reply:
Alright, will do that!